html/components/_toggle.scss

```diff
@@ -16,13 +16,6 @@
   text-decoration: $sprk-toggle-trigger-text-decoration;
   transition: $sprk-toggle-trigger-transition;
 
-  // Font considerations
-  color: $sprk-toggle-trigger-color;
```

Comment:
Did you delete these out of settings too?

Reply:
Nope! Sure didn't. I'll take care of that.